### PR TITLE
add org.mpdevil.mpdevil

### PR DESCRIPTION
**Pull request by SoongNoonien**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- access to the xdg-music directory to load album covers
- access to `org.freedesktop.Notifications` to send desktop notifications on title change
- [x] All assets referenced in the manifest are redistributible by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** http://mpdevil.org
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

Mpdevil is a small GTK music browser for MPD written in python.

### CODE REVIEW

**Comment by SoongNoonien** [comment]
bot, build org.mpdevil.mpdevil

**Comment by SoongNoonien** [comment]
bot, build org.mpdevil.mpdevil

**Comment by SoongNoonien** [comment]
bot, build org.mpdevil.mpdevil

File: org.mpdevil.mpdevil.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "app-id": "org.mpdevil.mpdevil",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.38",
+  "sdk": "org.gnome.Sdk",
+  "command": "mpdevil",
+  "finish-args": [
+    "--share=ipc",
```
Comment (barthalion) [inline org.mpdevil.mpdevil.json:8]:
```suggestion
```

AFAIK `fallback-x11` includes this permission.

Comment (SoongNoonien) [reply org.mpdevil.mpdevil.json:8]: OK, thanks for the tip.

Comment (dgcampea) [reply org.mpdevil.mpdevil.json:8]: > AFAIK `fallback-x11` includes this permission.

@barthalion  Does it? I think I've asked about this before and was told that it doesn't and the docs don't mention this at all.
Taking a quick glance at flatpak/flatpak, I'm unsure if this is the case (though I'm not familiar with flatpak code at all):
- https://github.com/flatpak/flatpak/blob/412c15772f794fd300991a91f2e0fe51f615bf79/common/flatpak-context.c#L967
- https://github.com/flatpak/flatpak/blob/412c15772f794fd300991a91f2e0fe51f615bf79/portal/flatpak-portal.c#L1027
  This one seems to imply the opposite?

**Pull request closed without merging**